src/main/java/org/zalando/nakadi/controller/ExceptionHandling.java

```diff
@@ -156,4 +157,12 @@ private String generateErrorTraceId() {
         return Responses.create(Response.Status.SERVICE_UNAVAILABLE, exception.getMessage(), request);
     }
 
+    @ExceptionHandler(DbWriteOperationsBlockedException.class)
+    public ResponseEntity<Problem> handleDbWriteOperationsBlockedException(
+            final DbWriteOperationsBlockedException exception, final NativeWebRequest request) {
+        LOG.error(exception.getMessage());
```

Comment:
I think it can be logged with WARN as this is situation that we create ourselves and we are aware of it, so it's not really an error.